addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -1741,6 +1741,50 @@ options.registry.collapse = options.Class.extend({
     },
 });
 
+options.registry.masonry_display = options.Class.extend({
+    xmlDependencies: (options.Class.xmlDependencies || []).concat(
+        ['/website/static/src/snippets/s_masonry_block/s_masonry_block.xml']
+    ),
+
+    /**
+     * @override
+     */
+    _computeWidgetState: function (methodName, params) {
+        if(methodName === 'editLayout') {
+            return params.variable || "default";
+        }
+        return this._super(...arguments);
+    },
```

Comment:
Private methods should be under a private section, something like:

```js
//--------------------------------------------------------------------------
// Private
//--------------------------------------------------------------------------

/**
 * @override
 */
_computeWidgetState(){...},
```

See existing options for that. The private section should be below the Options section that should contain your editLayout function